frontEnd/index.html

```diff
@@ -7,5 +7,20 @@
 </head>
 <body>
     
+<<<<<<< Updated upstream
+=======
+    <work>
+        
+        This is from the Work PC
+
+        When can I see this on the other side (Lenono PC)?
+
+        Why does this not merge with the existing main?
+
+        Is it because it is a fork?
+
```

Comment:
What is happening here?

It seems I can add comments to a project.

Reply:
I was testing to see if the main user can do edits on the existing code before committing.